components/pages/edit/ProposalEditForm.tsx

```diff
@@ -229,42 +197,9 @@ export default function ProposalEditForm({ space }: { space: string }) {
                   <Controller
                     name="proposal.body"
                     control={control}
-                    render={({ field: { onChange, onBlur, value, ref } }) =>
-                      <Editor
-                        apiKey={process.env.NEXT_PUBLIC_TINY_KEY || 'no-api-key'}
-                        onInit={(evt, editor) => editor.setContent(metadata.loadedProposal?.body || TEMPLATE)}
-                        initialValue={metadata.loadedProposal?.body || TEMPLATE}
-                        value={value}
-                        onEditorChange={(newValue, editor) => {
-                          if (!cacheModalIsOpen) {
-                            setProposalCache({ version: CACHE_VERSION, title: getValues("proposal.title") || "", body: newValue, timestamp: getUnixTime(new Date()) });
-                          }
-                          onChange(newValue);
-                        }}
-                        init={{
-                          height: 500,
-                          plugins: [
-                            'advlist', 'autolink', 'lists', 'link', 'image', 'preview',
-                            'anchor', 'searchreplace', 'code', 'fullscreen',
-                            'insertdatetime', 'table', 'code', 'help', 'wordcount',
-                            'image', 'autosave', 'template'
-                          ],
-                          toolbar: 'restoredraft undo redo | template blocks | ' +
-                            'image link table | bold italic forecolor | bullist numlist outdent indent | ' +
-                            'preview removeformat | help',
-                          menubar: false,
-                          block_unsupported_drop: false,
-                          images_upload_handler: imageUpload,
-                          init_instance_callback: fileDrop,
-                          content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }',
-                          autosave_restore_when_empty: true,
-                          templates: [
-                            { title: 'Proposal template', description: 'Default for most proposal', content: TEMPLATE }
-                          ],
-                          relative_urls: false,
-                          browser_spellcheck: true,
-                        }}
-                      />
+                    defaultValue={metadata.loadedProposal?.body || TEMPLATE}
+                    render={({ field: { onChange } }) =>
+                      <TextEditor onChange={onChange} initialValue={metadata.loadedProposal?.body || TEMPLATE}/>
```

Comment:
it should setProposalCache inside onChange which cached the changes in localstorage, let's keep it consistent.
```
onEditorChange={(newValue, editor) => {
  if (!cacheModalIsOpen) {
    setProposalCache({ version: CACHE_VERSION, title: getValues("proposal.title") || "", body: newValue, timestamp: getUnixTime(new Date()) });
  }
  onChange(newValue);
}}
```

Reply:
![image](https://github.com/nance-eth/nance-interface/assets/83923290/e3dad913-26fb-474f-9d10-8495f5ab80af)